assets/src/activation-notice/app/components/step1.js

```diff
@@ -19,17 +19,12 @@
  */
 import styled from 'styled-components';
 import { useCallback } from 'react';
-
-/**
- * WordPress dependencies
- */
-import { __, _x } from '@wordpress/i18n';
+import { __, _x, TranslateWithMarkup } from '@web-stories-wp/i18n';
```

Comment:
This is not desired. `activation-notice` and the `story-embed-block` should not use this package as they are WordPress-specific.